README.md

```diff
@@ -30,6 +21,26 @@ The Ja Rule codebase is licensed under the
 The unit-testing code & mocks are licenced under the
 [GPL](http://www.gnu.org/licenses/gpl.html).
 
+The hardware designs and the documentation is licensed under the
+[Creative Commons BY-SA](https://creativecommons.org/licenses/by-sa/3.0/us/).
+
+## Directory Layout
+
+```
+├── Bootloader  # The DFU bootloader
+│   └── firmware
+│       ├── Bootloader.X  # Bootloader MPLAB X project
+│       ├── src  # Bootloader source code
+├── common  # Common code shared between the bootloader and application.
+├── firmware  # The main DMX/RDM application
+│   ├── ja-rule.X
+│   └── src
+├── linker       # linker scripts for the bootloader & application
+├── tests        # Unit tests
+├── tools        # tools to upgrade the firmware on the device.
+└── user_manual  # The user manual documentation
```

Comment:
I'd say documentation is redundant here, unless it's the instructions for how to write or use the user manual; the user manual manual.